### PR TITLE
Removing duplicate definition

### PR DESCRIPTION
**Pull request by bgribaudo**
Rule _field-name_ is defined twice. Removing one of the definitions.

### CODE REVIEW

File: query-languages/m/m-spec-consolidated-grammar.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: M Language Consolidated Grammar 
 description: Describes all of the grammar associated with the Power Query M formula language
 ms.topic: conceptual
-ms.date: 9/15/2023
+ms.date: 2/2/24
```
Comment (Jak-MS) [inline query-languages/m/m-spec-consolidated-grammar.md:5]:
```suggestion
ms.date: 0202/2024
```

**Comment by Jak-MS** [comment]
@DougKlopfenstein 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: query-languages/m/m-spec-consolidated-grammar.md [hunk 2]
```diff
@@ -384,9 +384,6 @@ _field-list:<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;field_  `,`  _field-list<br/>
 field:<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;field-name_  `=`  _expression<br/>
-field-name:<br/>
```
Comment (ehrenMSFT) [inline query-languages/m/m-spec-consolidated-grammar.md:387]: It seems like it might make more sense to leave this one, and remove the one below.

**Review by ehrenMSFT — CHANGES_REQUESTED** [review]

File: query-languages/m/m-spec-consolidated-grammar.md (hunk 2 above)
Comment (bgribaudo) [reply query-languages/m/m-spec-consolidated-grammar.md]: Sounds good. Swapped.

**Comment by DougKlopfenstein** [comment]
#sign-off

**Pull request merged**